CameraXBasic/build.gradle

```diff
@@ -18,17 +20,17 @@
 
 buildscript {
     // Top-level variables used for versioning
-    ext.kotlin_version = '+'
+    ext.kotlin_version = KotlinVersion.CURRENT
```

Comment:
How is this better / different than '+'?

Reply:
none I can think of :/ 
reverted this change